os/binfmt/binfmt_exec.c

```diff
@@ -178,7 +179,7 @@ int exec(FAR const char *filename, FAR char *const *argv, FAR const struct symta
 	ret = binfmt_copyargv(bin, argv);
 	if (ret < 0) {
 		errcode = -ret;
-		berr("ERROR: Failed to copy argv[]: %d\n", errcode);
+		berr("%s #3: %d\n" ,clog_message_str[CMN_LOG_FAILED_OP], errcode);
```

Comment:
remove # numbers and keep some valid string. take care of # value in all the places.

Reply:
Ok, I will recheck and change.